stevenruidigao/gitlab-ce
# [Snyk] Security upgrade ruby from 2.5-stretch to 2.6.8-stretch

<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br />Keeping your Docker base image up-to-date means you’ll benefit from security fixes in the latest version of your chosen image.

#### Changes included in this PR 


- qa/Dockerfile

We recommend upgrading to `ruby:2.6.8-stretch`, as this image has only 654 known vulnerabilities. To do this, merge this pull request, then verify your application still works as expected.



Some of the most important vulnerabilities in your base image include:

| Severity                                                                                                                 | Priority Score / 1000  | Issue                                                                     | Exploit Maturity      |
| :------:                                                                                                                 | :--------------------  | :----                                                                     | :---------------      |
| ![high severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/h.png "high severity")   | **614**  | Loop with Unreachable Exit Condition (&#x27;Infinite Loop&#x27;) <br/>[SNYK-DEBIAN9-LIBXML2-429485](https://snyk.io/vuln/SNYK-DEBIAN9-LIBXML2-429485)   | No Known Exploit   |
| ![high severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/h.png "high severity")   | **614**  | Out-of-bounds Read <br/>[SNYK-DEBIAN9-OPENSSL-1569399](https://snyk.io/vuln/SNYK-DEBIAN9-OPENSSL-1569399)   | No Known Exploit   |
| ![high severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/h.png "high severity")   | **678**  | Loop with Unreachable Exit Condition (&#x27;Infinite Loop&#x27;) <br/>[SNYK-DEBIAN9-OPENSSL-2426305](https://snyk.io/vuln/SNYK-DEBIAN9-OPENSSL-2426305)   | No Known Exploit   |
| ![critical severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/c.png "critical severity")   | **714**  | Buffer Overflow <br/>[SNYK-DEBIAN9-PYTHON27-1063180](https://snyk.io/vuln/SNYK-DEBIAN9-PYTHON27-1063180)   | No Known Exploit   |
| ![high severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/h.png "high severity")   | **614**  | Resource Exhaustion <br/>[SNYK-DEBIAN9-PYTHON35-1570179](https://snyk.io/vuln/SNYK-DEBIAN9-PYTHON35-1570179)   | No Known Exploit   |



---

**Note:** _You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs._

For more information: <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiJlOTM0MDI2OS0yMWM2LTRiODAtODJmZS02MTE1MWZlZGMyMjgiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6ImU5MzQwMjY5LTIxYzYtNGI4MC04MmZlLTYxMTUxZmVkYzIyOCJ9fQ==" width="0" height="0"/>
🧐 [View latest project report](https://app.snyk.io/org/stevenruidigao/project/eac422ed-e0fc-442f-9cfd-4deaa9bceaf3?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)

🛠 [Adjust project settings](https://app.snyk.io/org/stevenruidigao/project/eac422ed-e0fc-442f-9cfd-4deaa9bceaf3?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)

[//]: # 'snyk:metadata:{"prId":"e9340269-21c6-4b80-82fe-61151fedc228","prPublicId":"e9340269-21c6-4b80-82fe-61151fedc228","dependencies":[{"name":"ruby","from":"2.5-stretch","to":"2.6.8-stretch"}],"packageManager":"dockerfile","projectPublicId":"eac422ed-e0fc-442f-9cfd-4deaa9bceaf3","projectUrl":"https://app.snyk.io/org/stevenruidigao/project/eac422ed-e0fc-442f-9cfd-4deaa9bceaf3?utm_source=github&utm_medium=referral&page=fix-pr","type":"auto","patch":[],"vulns":["SNYK-DEBIAN9-PYTHON27-1063180","SNYK-DEBIAN9-OPENSSL-2426305","SNYK-DEBIAN9-OPENSSL-1569399","SNYK-DEBIAN9-PYTHON35-1570179","SNYK-DEBIAN9-LIBXML2-429485"],"upgrade":["SNYK-DEBIAN9-LIBXML2-429485","SNYK-DEBIAN9-OPENSSL-1569399","SNYK-DEBIAN9-OPENSSL-2426305","SNYK-DEBIAN9-PYTHON27-1063180","SNYK-DEBIAN9-PYTHON35-1570179"],"isBreakingChange":false,"env":"prod","prType":"fix","templateVariants":["updated-fix-title","priorityScore"],"priorityScoreList":[714,678,614,614,614]}'

---

**Learn how to fix vulnerabilities with free interactive lessons:**

 🦉 [Learn about vulnerability in an interactive lesson of Snyk Learn.](https://learn.snyk.io?loc&#x3D;fix-pr)
